crowbar_framework/vendor/assets/stylesheets/variables.scss

```diff
@@ -340,6 +340,7 @@ $state-warning-border: darken(adjust-hue($state-warning-bg, -10), 5%);
 
 $state-danger-text: #a94442;
 $state-danger-bg: #f2dede;
+$state-danger-pre-bg: lighten($state-danger-bg, 5%);
```

Comment:
Move this to `bootstrap/_variables.scss`. `variables.scss` doesn't seem to be included anywhere.

Reply:
Thanks, fixed.